web/templates/stream_settings/stream_settings.hbs

```diff
@@ -53,15 +53,7 @@
                   rendered_description=rendered_description
                   }}
             </div>
-            <div class="stream-permissions settings-subsection-parent" id="stream_permission_settings">
-                <div class="subsection-header">
-                    <h3 class="stream_setting_subsection_title">{{t "Channel permissions" }}
-                    </h3>
-                    {{> ../settings/settings_save_discard_widget section_name="stream-permissions" }}
-                </div>
-
-                <div class="stream-permissions-warning-banner"></div>
-
+            <div class="stream-permissions" id="stream_permission_settings">
```

Comment:
Can we skip this adding this `div`? There is already a containing `div` in `stream_types`.